packages/actionmenu/package.json

```diff
@@ -15,15 +15,10 @@
     "build:watch": "npm run build:js -- --watch",
     "prepublish": "npm run build",
     "storybook": "start-storybook -p 6006",
-    "test": "jest",
+    "test": "../../node_modules/.bin/jest",
```

Comment:
we could simplify this if we used something like https://github.com/nicojs/node-link-parent-bin